### PR TITLE
8263417: Add Extract integer nodes implementation for Arm SVE

### PR DESCRIPTION
**Pull request by Wanghuang-Huawei**
* Add Extract integer nodes implementation for Arm SVE, like ExtractB/ExtractS/ExtractI/ExtractL
* add `sve_lasta(Register, SIMD_RegVariant, PRegister,  FloatRegister)` & `sve_lastb`

---------
### Progress
- [x] Change must not contain extraneous whitespace
- [x] Change must be properly reviewed

### Issue
 * [JDK-8263417](https://bugs.openjdk.java.net/browse/JDK-8263417): Add Extract integer nodes implementation for Arm SVE

### Reviewers
 * [Ningsheng Jian](https://openjdk.java.net/census#njian) (@nsjian - Committer)

### Contributors
 * Wang Huang `<whuang@openjdk.org>`
 * He Xuejin `<hexuejin2@huawei.com>`
 * Ai Jiaming `<aijiaming1@huawei.com>`

### Download
`$ git fetch https://git.openjdk.java.net/panama-vector pull/48/head:pull/48`
`$ git checkout pull/48`

### CODE REVIEW

File: src/hotspot/cpu/aarch64/aarch64_sve.ad [hunk 1]
```diff
@@ -2991,3 +2986,78 @@ instruct vcvtDtoB(vReg dst, vReg src, vReg tmp)
...
+  match(Set dst (ExtractB src idx));
+  effect(TEMP tmp, TEMP pTmp, KILL cr);
+  ins_cost(2 * SVE_COST);
+  format %{ "movzw $tmp, $idx\n\t"
+            "sve_whilele $pTmp, B, zr, $tmp\n\t"
+            "sve_lastb $dst, B, $pTmp, $src\n\t"
+            "sbfmw $dst, $dst, 0U, 7U\t# extract from vector(B)" %}
+  ins_encode %{
+    __ movzw(as_Register($tmp$$reg), (int)($idx$$constant));
+    __ sve_whilele(as_PRegister($pTmp$$reg), __ B, zr, as_Register($tmp$$reg));
```
Comment (nsjian) [inline src/hotspot/cpu/aarch64/aarch64_sve.ad:3004]: I think we can use rscratch1 here to save one register. See https://github.com/openjdk/panama-vector/pull/49

Comment (Wanghuang-Huawei) [reply src/hotspot/cpu/aarch64/aarch64_sve.ad:3004]: OK. Thank you for your review.I will fix it in my next patch. For other `$tmp` in `adfile`, I will create a new issue (JDK-8263578) and try to replace them.

**Review by nsjian — APPROVED** [review]

**Pull request closed without merging**